client/src/components/relatedProducts/CardButtons.jsx

```diff
@@ -176,4 +204,5 @@ export { OnCardClick };
 export { StarButton };
 export { PlusButton };
 export { LeftArrow };
-export { RightArrow };
\ No newline at end of file
+export { RightArrow };
+export { DetermineAction };
```

Comment:
For consideration: As you clean up for final rendering, could be helpful to put these all into a module.exports larger object! Just a thought, this works too!